### PR TITLE
[13.0][MIG]purchase_invoicing_no_zero_line: Migration to 13.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT34985

ping @pedrobaeza @victoralmau

### CODE REVIEW

File: purchase_invoicing_no_zero_line/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2019 Digital5 S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Purchase invoicing no zero line",
+    "summary": "Avoid creation of zero quantity invoice lines from purchase",
+    "version": "13.0.1.0.0",
+    "category": "Purchase Management",
+    "author": "Digital5 S.L., Odoo Community Association (OCA)",
+    "license": "AGPL-3",
```
Comment (victoralmau) [inline purchase_invoicing_no_zero_line/__manifest__.py:9]:
```suggestion
    "license": "AGPL-3",
    "website": "https://github.com/OCA/account-invoicing",
```

**Review by victoralmau — APPROVED** [review]
Code review OK

**Review by pedrobaeza — APPROVED** [review]
You can squash together initial commits from the same author

**Comment by CarlosRoca13** [comment]
Changes done @victoralmau @pedrobaeza

**Pull request merged**